### PR TITLE
Fixed example and index-page generation

### PR DESCRIPTION
**Pull request by hblitza**
### Description:

Example-indexpage: fixed path to css-file.
Landing index page: added some text and example images

### CODE REVIEW

File: assets/gh-index.html [hunk 1]
```diff
@@ -1,113 +1,85 @@
...
+	<meta charset="utf-8">
+	<meta content="width=device-width, initial-scale=1" name="viewport">
+	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js">
+	</script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js">
+	</script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js">
+	</script>
+	<link href="assets/landing_page.css" rel="stylesheet" type="text/css">
```
Comment (dnlkoch) [inline assets/gh-index.html]: Please check indentation.

File: assets/gh-index.html [hunk 2]
```diff
@@ -1,113 +1,85 @@
...
+					<h3>Check out <a href="examples/index.html">examples</a> and <a href="./wiki/Feature-List">feature list</a> for full functionality of react-geo.</h3><br>
+					<img class="rounded img-exp" src="assets/geoutil.png" style="vertical-align:middle" width="286px"> <span style="font-size: 22px;">GeometryUtil features<br>
+					<span style="font-size: 22px;">Map components</span><img class="rounded img-exp" src="assets/distance.png"></span>
+					<div class="carousel-caption"></div>
+				</div><!-- Left and right controls -->
+				 <a class="left carousel-control" data-slide="prev" href="#myCarousel" role="button"><span aria-hidden="true" class="glyphicon glyphicon-chevron-left"></span> <span class="sr-only">Previous</span></a> <a class="right carousel-control" data-slide="next" href="#myCarousel" role="button"><span aria-hidden="true" class="glyphicon glyphicon-chevron-right"></span> <span class="sr-only">Next</span></a>
+			</div>
+		</div><!-- Footer -->
+		<footer class="container-fluid-footer bg-4 text-center"></footer>
+	</div>
```
Comment (dnlkoch) [inline assets/gh-index.html]: Please check indentation of the `<body>` contents.

File: assets/gh-index.html [hunk 3]
```diff
@@ -1,113 +1,85 @@
...
+		</div>
+	</nav><!-- First Container -->
+	<div class="container-fluid text-center">
+		<div class="carousel slide" id="myCarousel">
+			<!-- Wrapper for slides -->
+			<div class="carousel-inner" role="listbox">
+				<div class="text-center item active">
+					<img class="img-responsive img-circle margin" src="assets/logo.svg" style="display:inline" width="200">
+					<h3>A set of geo related modules.<br>
+					To use in combination with <a href="https://reactjs.org/">react</a>, <a href="https://ant.design/">antd</a> and <a href="https://openlayers.org">ol</a>.</h3>
```
Comment (dnlkoch) [inline assets/gh-index.html]: Better as one single sentence: `…modules.<br>	To…` -> `…modules to…`

File: assets/gh-index.html [hunk 4]
```diff
@@ -1,113 +1,85 @@
...
+					<div class="carousel-caption"></div>
+				</div>
+				<div class="item">
+					<div class="item2">
+						<h3>Create your own WebGIS easily with the power of reactjs, andt and Openlayers</h3>
```
Comment (dnlkoch) [inline assets/gh-index.html]: Some typos:
  * `reactjs` --> `React`
  * `andt` --> `antd`
  * `Openlayers` --> `OpenLayers`

**Comment by hblitza** [comment]
`__VERSION__`  placeholder in the docs-link. That works for the gh-index.html. But does it work within the generation of the example page (_index.bs_)?

**Comment by dnlkoch** [comment]
No, not out of the box. The current `__VERSION__` will be replace inside the `update-gh-pages.sh` script after all files have been generated. See [here](https://github.com/terrestris/react-geo/blob/master/.ci/update-gh-pages.sh#L53).

**Comment by marcjansen** [comment]
#484 related? Good work here, friends.

**Pull request merged**